Tools/Scripts/hooks/pre-push

```diff
@@ -237,22 +230,35 @@ def security_level_for_commit(commit, remotes, name_to_remote=None):
         if secondary:
             secondary = UNPACK_SECONDARY_RE.match(secondary).groups()[0]
         for candidate in [primary, secondary]:
-            original_remotes = remotes_for(candidate)
+            original_remotes = remotes_for(candidate.rstrip('.')) if candidate else []
```

Comment:
`candidate.rstrip('.')`: What is this doing? Is it possible to move the `.` elision into the regular expression this variable comes from?

nit: I think this is more readable:
```python3
        for candidate in [primary, secondary]:
            if not candidate: 
                continue
            original_remotes = remotes_for(candidate.rstrip('.'))
```

or even:
```python3
        for candidate in filter(bool, (primary, secondary)):
            original_remotes = remotes_for(candidate.rstrip('.'))
```